examples/vhost-php.conf

```diff
@@ -1,28 +1,22 @@
-{{LOG_DIR=/var/log/apache2}}
-{{RUN_DIR=/var/run/php-fpm}}
-{{FCGI=$RUN_DIR/$DOMAIN.fcgi}}
-{{SOCKET=$RUN_DIR/$DOMAIN.sock}}
-{{EMAIL=$USER@$DOMAIN}}
-{{DOC_ROOT=/home/$USER/sites/$DOMAIN/htdocs}}
 <VirtualHost *:80>
-  ServerAdmin {{EMAIL}}
-  ServerName {{DOMAIN}}
-  ServerAlias www.{{DOMAIN}}
+  ServerAdmin nobody@example.com
```

Comment:
Is it really correct that this file should not contain placeholders anymore?

Reply:
Yes, it seems so. Please see `../test.sh`. Those `vhost-php*.conf` files are now both example and part of test procedure.  The author decided to use `vhost-php.tpl.conf` as actual template and `vhost-php.conf` as "expected result".